packages/aws-cdk-lib/aws-rds/lib/cluster.ts

```diff
@@ -1164,6 +1178,13 @@ abstract class DatabaseClusterNew extends DatabaseClusterBase {
       throw new ValidationError('serverlessV2MinCapacity & serverlessV2MaxCapacity must be in 0.5 step increments, received '+
       `min: ${this.serverlessV2MaxCapacity}, max: ${this.serverlessV2MaxCapacity}`, this);
     }
+
```

Comment:
The [documentation](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless-v2-auto-pause.html#auto-pause-prereqs) states the following pre-requisites:

> * If you're using Aurora PostgreSQL, the database engine must be running at least version 16.3, 15.7, 14.12, or 13.15.
> * If you're using Aurora MySQL, the database engine must be running version 3.08.0 or higher.

I think it would be good to add this validation here as well, so users know what is needed for the configuration to actually be applied correctly.